From af5d91967ffabd09b2a8be3afd3544c875bdcc27 Mon Sep 17 00:00:00 2001
From: Allen Xu <b45815@freescale.com>
Date: Fri, 14 Mar 2014 15:11:37 -0500
Subject: [PATCH] ENGR00303629-1: ARM: dts: imx6sx-sdb:  support NOR Flash
 SF25FL128S

add Spansion SF25FL128S in sdb dts file.

Signed-off-by: Allen Xu <b45815@freescale.com>
(cherry picked from commit b7efdb815a9cfb63b39823226316806aced21b33)
---
 arch/arm/boot/dts/imx6sx-sdb.dts | 22 ++++++++++++++++++++++
 1 file changed, 22 insertions(+)

diff --git a/arch/arm/boot/dts/imx6sx-sdb.dts b/arch/arm/boot/dts/imx6sx-sdb.dts
index 1affbbc..922d0d5 100644
--- a/arch/arm/boot/dts/imx6sx-sdb.dts
+++ b/arch/arm/boot/dts/imx6sx-sdb.dts
@@ -473,3 +473,25 @@
 		};
 	};
 };
+
+&qspi2 {
+	pinctrl-names = "default";
+	pinctrl-0 = <&pinctrl_qspi2_1>;
+	status = "okay";
+
+	flash0: s25fl128s@0 {
+		#address-cells = <1>;
+		#size-cells = <1>;
+		compatible = "spansion,s25fl128s";
+		spi-max-frequency = <66000000>;
+		reg = <0>;
+	};
+
+	flash1: s25fl128s@1 {
+		#address-cells = <1>;
+		#size-cells = <1>;
+		compatible = "spansion,s25fl128s";
+		spi-max-frequency = <66000000>;
+		reg = <1>;
+	};
+};
-- 
1.8.0

